charts/incubator/remix-plex/values.yaml

```diff
@@ -0,0 +1,76 @@
+image:
+  repository: ghcr.io/lostb1t/replex
+  pullPolicy: IfNotPresent
+  tag: 0.20.0@sha256:f6cdeb27e7aaad36317c0fdf3cce06bbc96189cb2aacd5d60a98878fc6e8c467
+
+securityContext:
+  container:
+    readOnlyRootFilesystem: false
+    runAsUser: 0
+    runAsGroup: 0
+
+service:
+  main:
+    ports:
+      main:
+        protocol: http
+        targetPort: 80
+        port: 32401
+
+replex:
+  host: ""
+  hero_rows:
+    - home.movies.recent
+    - movies.recent
+    - movie.recentlyadded
+    - movie.topunwatched
+    - movie.recentlyviewed
+    - hub.movie.recentlyreleased
+    - movie.recentlyreleased
+    - home.television.recent
+    - tv.recentlyadded
+    - tv.toprated
+    - tv.inprogress
+    - tv.recentlyaired
+  exclude_watched: false
+  disable_continue_watching: false
+  disable_user_state: false
+  disable_leaf_count: false
+  force_max_quality: false
+  force_direct_play_for: false
```

Comment:
This seems to must be a string instead. 
![image](https://github.com/truecharts/charts/assets/47820033/74177dc7-24b7-4ce4-9273-605cc55005c1)
https://github.com/lostb1t/replex/blob/a5dc266588739be2a03728669ab988d263168d19/src/config.rs#L90C9-L90C30